src/Modules/CalcDefence.lua

```diff
@@ -225,8 +238,7 @@ function calcs.reducePoolsByDamage(poolTable, damageTable, actor)
 	end
 
 	return {
-		FrostShieldLife = frostShield,
-		SoulLink = soulLink,
+		alliesTakenBeforeYou = alliesTakenBeforeYou,
```

Comment:
```suggestion
		AlliesTakenBeforeYou = alliesTakenBeforeYou,
```
The rest of the fields in this return type are capitalised consider doing the same here.